integration-test/test-pages/broker-protection/actions/results-not-found-invalid.json

```diff
@@ -0,0 +1,52 @@
+{
+    "state": {
+      "action": {
+        "actionType": "extract",
+        "selector": ".search-item-invalid",
+        "noResultsSelector": ".none",
+        "retry": {
+          "environment": "web",
+          "interval": { "ms": 1000 },
+          "maxAttempts": 1
+        },
```

Comment:
@brianhall I added this as a way to opt-out of the `extract` actions hard-coded retry logic. we should address this later if it becomes more annoying.

Reply:
Ah, didn’t occur to me to set the retry lower than the playwright timeout threshold. That’s a good workaround!